### ISSUE TITLE
NDCGLoss

### ISSUE THREAD

**Issue by celsofranssa** [KevinMusgrave/pytorch-metric-learning#675]
Since metric learning is very useful for ranking tasks, I'd like to suggest including the [ApproxNDCGLoss](https://www.tensorflow.org/ranking/api_docs/python/tfr/keras/losses/ApproxNDCGLoss)  and other ranking losses such as [L2R losses](https://pytorchltr.readthedocs.io/en/stable/loss.html) using the PML approach.
